Scripts/RemediationScripts/Remediate-SetClassicFrontDoorMinTLSVersion.ps1

```diff
@@ -0,0 +1,1020 @@
+                    Write-Host $([Constants]::SingleDashLine)
+                    
+                    $resource = Enable-AzFrontDoorCustomDomainHttps -ResourceGroupName $resourceGroupName -FrontDoorName $resourceName -FrontendEndpointName $hostName -MinimumTlsVersion $requiredMinTLSVersion
+                    
+                    if($resource.MinimumTlsVersion -eq $requiredMinTLSVersion)
+                    {
+                        $frontdoor.isMinTLSVersionSetOnCustomDomainPostRemediation = $true
+                        $frontdoor.minimumTlsVersion = $resource.MinimumTlsVersion
+                        $customDomainsRemediated += $frontdoor
+                        Write-Host "Minimum required TLS version for Front Door endpoint has been set successfully for :" $hostName -ForegroundColor $([Constants]::MessageType.Update)
+                        Write-Host $([Constants]::SingleDashLine)
+                    }
+                    else
+                    {
+                        $frontDoorsSkipped += $frontdoor
+                        $logResource = @{}
+                        $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                        $logResource.Add("ResourceName",($_.ResourceName))
+                        $logResource.Add("Reason", "Error while setting the minimum required TLS version. Skipping this Front Door endpoint :" + $hostName )
+                        $logSkippedResources += $logResource
+                        Write-Host "Error while setting the minimum required TLS version for this Front Door Endpoint :" $hostName -ForegroundColor $([Constants]::MessageType.Error)
+                        Write-Host "Skipping this Front Door endpoint." -ForegroundColor $([Constants]::MessageType.Warning)
+                        Write-Host $([Constants]::SingleDashLine)
+                        return
+                    }
+                }
+                catch
+                {
+                    $frontDoorsSkipped += $frontdoor
+                    $logResource = @{}
+                    $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                    $logResource.Add("ResourceName",($_.ResourceName))
+                    $logResource.Add("Reason", "Error while setting the minimum required TLS version on the custom domain. Skipping this Front Door.")
+                    $logSkippedResources += $logResource
+                    Write-Host "Error while setting the minimum required TLS version on the custom domain of Front Door." -ForegroundColor $([Constants]::MessageType.Error)
+                    Write-Host $([Constants]::SingleDashLine)
+                    return
+                }
+            }
+        
+            if ($($frontDoorsSkipped | Measure-Object).Count -eq 0)
+            {
+                #$frontdoor.isMinTlsVersionSetOnCustomDomain = $true
```

Comment:
what's the use case of this logging here.

Reply:
Removed logging from there